popup.js

```diff
@@ -0,0 +1,126 @@
+       <li><span class="htcjmodal">${projectCards[0].technologies[2]}</span></li>
+   </ul>   
+   <ul class="techlistmodal2">
+       <li><span class="htcjmodal">${projectCards[0].technologies[3]} </span> </li>
+       <li><span class="htcjmodal">${projectCards[0].technologies[4]} </span></li>
+       <li><span class="htcjmodal">${projectCards[0].technologies[5]}</span></li>
+   </ul>     
+   <br>
+   <span><a class="abouts" href="https://baqar-abbas.github.io/Portfolio-Project-Final-version/" target="_blank">See Live<img class="seeliveicon1" src="seeliveicon.png"></a> </span> 
+   <span><a class="abouts" href="https://github.com/baqar-abbas/Portfolio-Project-Final-version/tree/add-popup-Modal" target="_blank">See Source<img class="gitbuttonicon1" src="gitbutton.png"></a> </span> 
+   `;
+
+    page.style.display = 'block';
+    background.style.display = 'block';
+  } else if (n === 2) {
+    selectdivmob.innerHTML = `<h2 class="popheading modalheading2">${projectCards[0].name2}</h2> <a href="javascript:;" onclick="popupclose()"><img class="crossmobile" src="cross1.png" alt="cross2" height="35px" width="35px"></a>
+     <p class="popp13 mobilep">${projectCards[0].specifics2[0]} <img class="dot" src="dot.png" alt="dot"> 
+                          <span class="popp13 col">${projectCards[0].specifics2[1]}</span> <img class="dot" src="dot.png" alt="dot"> 
+                           <span class="popp13 col">${projectCards[0].specifics2[2]}</span></p> 
+                           <img src="image1.png" alt="MobileImage" height="220px" width="311px">
+                           <p class="popp15">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's 
+                           standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make
+                            a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, 
+                            remaining essent</p> <ul class="techlist">
+                            <li><span class="htcj">${projectCards[0].technologies[0]}</span> </li>
+                            <li><span class="htcj">${projectCards[0].technologies[1]}</span></li>
+                            <li><span class="htcj">${projectCards[0].technologies[2]}</span></li>
+                        </ul>
+                        <span><a class="abouts paddingt" href="https://baqar-abbas.github.io/Portfolio-Project-Final-version/" target="_blank">See Live<img class="seeliveicon2" src="seeliveicon.png"></a> </span> 
+                        <span><a class="abouts paddingt" href="https://github.com/baqar-abbas/Portfolio-Project-Final-version/tree/add-popup-Modal" target="_blank">See Source<img class="gitbuttonicon2" src="gitbutton.png"></a> </span>`;
+    pagemob.style.display = 'block';
+  }
+  pagemob.style.display = 'block';
+  backgroundmobile.style.display = 'block';
+}
```

Comment:
- [ ] Kindly make sure to display only the information of each object from the array of projects, because you have added the hardcoded information (project description) while you need to add this part from the object of each object.

- [ ] Kindly make sure to iterate through the array of all objects because, at this moment, we can only see one pop-up card with the same information on all cards. Please make sure if we clicked project 3 we get the unique details of project 3.

- [ ] Kindly make sure to fix the image on the desktop version, at this moment the image is not visible.

Reply:
Thanks for your valuable remarks. I am putting this project description in objects and will display this project details information by accessing object. And right now I have put two use cases for displaying projects card one for displaying Desktop data and one for mobile. I can manage to show different project depending on which project user clicks on. 
I am incorporating your suggested changes within the project.

Reply:
I am adding other 6 use cases (3 for Desktop and 3 for Mobile) for displaying different data for different projects.

Reply:
Image is appearing properly screen shot attached below for your kind reference.
![image](https://user-images.githubusercontent.com/117091923/217865820-78c1bebf-712b-4fb5-9fad-7947c85cd73a.png)

Reply:
Suggested changes made, hardcoded information removed stored in array of objects and display projects details from array of objects. Screen shots attached below for your kind reference.
![image](https://user-images.githubusercontent.com/117091923/217867006-1d94e435-333d-4b47-abe4-14cda879cb7a.png)
![image](https://user-images.githubusercontent.com/117091923/217867233-001436d6-094e-4bf9-8a9a-7a96a6ab1daf.png)